src/app/pages/task-calendar/rsync/rsync-form/rsync-form.component.ts

```diff
@@ -173,9 +178,9 @@ export class RsyncFormComponent {
         type: 'checkbox',
         name: 'rsync_preserveattr',
         placeholder: T('Preserve extended attributes'),
-        tooltip: T('Both systems must support <a\
+        tooltip: T('Both systems <b>must</b> support <a\
```

Comment:
Needs to tell what it does first.
'Extended attributes are preserved, but both systems...'

Reply:
done